docs/pipeline-customization.md

```diff
@@ -84,35 +96,10 @@ metadata:
 spec:
   templates:
     job:
-      # Job spec
       ttlSecondsAfterFinished: 600 # numaflow defaults to 30
       backoffLimit: 5 # numaflow defaults to 20
-      # Pod metadata
-      metadata:
-        labels:
-          my-label-name: my-label-value
-        annotations:
-          my-annotation-name: my-annotation-value
-      # Pod spec
-      nodeSelector:
-        my-node-label-name: my-node-label-value
-      tolerations:
-        - key: "my-example-key"
-          operator: "Exists"
-          effect: "NoSchedule"
-      securityContext: {}
-      imagePullSecrets:
-        - name: regcred
-      priorityClassName: my-priority-class-name
-      priority: 50
-      serviceAccountName: my-service-account
-      affinity: {}
-      # Container
-      containerTemplate:
-        env:
-          - name: MY_ENV_NAME
-            value: my-env-value
-        resources:
-          limits:
-            memory: 500Mi
 ```
+
+## Vertex customization
+
+See `Component customization` described above.
```

Comment:
Is there an place to mention that it can also be done in pipeline level?

Reply:
I'm a bit confused, as this is for the pipeline level, do you mean modify this to mention it's for the pipeline-level?  At the top of the page in the `Pipeline Customization` section I try to describe things can be done in the controller configmap and vertexes can be customized individually, but may not be doing a great job with the wording.

Reply:
With this change, there are 3 places to customize the vertex template - configmap, `pipeline spec.template.vertex`, and `pipeline.spec.vertices[*].xxx`, I don't see the 2nd one is mentioned.

Reply:
The "component customization" section in this file and the comment in the example attempt to describe that.